test/libsolidity/SolidityEndToEndTest.cpp

```diff
@@ -7964,8 +7968,10 @@ BOOST_AUTO_TEST_CASE(state_variable_local_variable_mixture)
 		}
 	)";
 
-	compileAndRun(sourceCode);
-	ABI_CHECK(callContractFunction("a()"), encodeArgs(u256(2)));
+	ALSO_VIA_YUL(
```

Comment:
Accessing via member access to contract names seems to be unimplemented as of yet.